src/tests/Microsoft.PowerFx.Interpreter.Tests/LanguageTest.cs

```diff
@@ -63,6 +64,31 @@ public void GetLanguageForInvariantCulture()
             ConfigTests.RunOnIsolatedThread(_defaultCulture, TestDefaultCulture);
         }
 
+        [Theory]
+        [InlineData("1234,5678", "#.##0,00", "1.234,57", "vi-VN")]
+        [InlineData("1234,5678", "#.##0,00", "1.234,57", "pt-BR")]
```

Comment:
>pt-BR" [](http://example.com/codeflow?start=58&length=6)

pt-BR and vi-VN have the same separators; consider using a language that has a different one (such as fi-FI, which uses U+00A0 as the thousands grouping)

Reply:
add fi-FI.